core/workspace_svg.js

```diff
@@ -382,9 +401,22 @@ Blockly.WorkspaceSvg.prototype.resize = function() {
   if (this.scrollbar) {
     this.scrollbar.resize();
   }
+  this.updateScreenDependentCalculations_();  
```

Comment:
Wow. I botched my last rename of this function fairly badly.... Should be fixed now.